.pre-commit-config.yaml

```diff
@@ -62,6 +62,14 @@ repos:
       - id: isort
         name: isort (python)
 
+  - repo: https://github.com/myint/autoflake
+    rev: v1.4
+    hooks:
+      - id: autoflake
+        args:
+          - --in-place
+          - --remove-unused-variables
```

Comment:
Q: Could we also add unused imports? (also, I think we should also use recursive?)

Reply:
- Remove (python standard) libraries is used by default. Remove other libraries could raise unexpected errors I think

- yes, I will include recursive

Reply:
For 1 I think we can put a "noqa" like text next to the import.